`test_multi_notif_event_types` fails due to missing events

```
E           ocs_ci.ocs.exceptions.TimeoutExpiredError: Some expected events were not received by Kafka: {('LifecycleExpiration:DeleteMarkerCreated', 'versioned_expired/versioned_expired-3'), ('LifecycleExpiration:DeleteMarkerCreated', 'versioned_expired/versioned_expired-0'), ('LifecycleExpiration:DeleteMarkerCreated', 'versioned_expired/versioned_expired-2'), ('LifecycleExpiration:DeleteMarkerCreated', 'versioned_expired/versioned_expired-1'), ('LifecycleExpiration:DeleteMarkerCreated', 'versioned_expired/versioned_expired-4')}
```
Previously these events wouldn't show because of a bug: https://issues.redhat.com/browse/DFBUGS-1468
Now that it's fixed, they should be showing.

RP link for reference: https://url.corp.redhat.com/6493e29

--- a/tests/functional/object/mcg/test_bucket_notifications.py
+++ b/tests/functional/object/mcg/test_bucket_notifications.py
@@ -185,7 +185,6 @@ class TestBucketNotifications(MCGTest):
         bucket_factory,
         test_directory_setup,
         notif_manager,
-        jira_issue,
     ):
         """
         Test that various bucket notification events are received by Kafka
@@ -302,23 +301,10 @@ class TestBucketNotifications(MCGTest):
             for obj_key in prefix_to_obj[prefix]:
                 expected_events.add((event, os.path.join(prefix, obj_key)))
 
-        if jira_issue("DFBUGS-1468"):
-            logger.warning(
-                (
-                    "Not testing the LifecycleExpiration:DeleteMarkerCreated"
-                    " event due to DFBUGS-1468"
-                )
-            )
-            expected_events = {
-                event
-                for event in expected_events
-                if "LifecycleExpiration:DeleteMarkerCreated" not in event
-            }
-
         delta = set()
         try:
             for raw_received_events in TimeoutSampler(
-                timeout=120,
+                timeout=600,
                 sleep=5,
                 func=notif_manager.get_events,
                 topic=topic,

